docs/usage.md

```diff
@@ -263,4 +268,34 @@ Following section explains how to get a list of closed change requests (CRs) sin
 
 2. Click **Apply Latest Content** tile under the **Production** tab.
 
-3. Click **Fetch Latest Changes** button from the lower-left of the screen to get the list of issues resolved since the last applied change.
\ No newline at end of file
+3. Click **Fetch Latest Changes** button from the lower-left of the screen to get the list of issues resolved since the last applied change.
+
+## Known Issues
+
+Renaming a record, after raising the PR, creates a duplicate record with the same UUID when the changes are pushed through the PR. This action results in the import process (*Apply Latest Content*) failing.
+
+### Error
+
+```
+409 Conflict response:
+{"type":"UniqueConstraintViolationException","message":"Failed to create the record.
+The system administrator has define (truncated...)
+```
+
+### Workaround
+
+After the records have been renamed and before pushing the changes for review, delete the old records. Refer to [Deleting files in a repository - GitHub](https://docs.github.com/en/repositories/working-with-files/managing-files/deleting-files-in-a-repository) for deleting the file from GitHub.
```

Comment:
Suggestion: After the records have been renamed and before pushing the changes for review, delete the old file/folder from the PR. Refer to.....

Reply:
Changed